app/views/batches/_cherrypick_single_worksheet.html.erb

```diff
@@ -92,6 +92,7 @@
 </div>
 
 <div id="footer">
-  Created: <%= batch.updated_at.strftime("%I:%M %p on %A %d %B, %Y") %> for <%= batch.user.login %> -
+  v = receptacle volume; b = buffer volume<br>
+  Created: <%= batch.updated_at.strftime("%I:%M %p on %A %d %B, %Y") %> for <%= batch.user.login %><br>
```

Comment:
Great idea to add this, thank you. 👍 

I would go for 'picking volume' instead of 'receptacle volume'. 

'Receptacle volume' is ambiguous as to whether it's talking about the source well or the destination well - and in fact it's neither - it's the volume that is going to be taken out of the source well to be added to the destination well (but might not be the final destination well volume because of added buffer).

Is that correct / do you agree?

Also, in your screenshot, you've added units, but I don't see them here.